lib/components/call_page.dart

```diff
@@ -0,0 +1,380 @@
+              ),
+              RawMaterialButton(
+                // if user on the other end hung up, then call will end on both sides
+                onPressed: () => _onCallEnd(context),
+                child: Icon(
+                  Icons.call_end,
+                  color: Colors.white,
+                  size: 35.0,
+                ),
+                shape: CircleBorder(),
+                elevation: 2.0,
+                fillColor: Colors.redAccent,
+                padding: const EdgeInsets.all(15.0),
+              ),
+              RawMaterialButton(
+                onPressed: _onToggleMuteAudio,
+                child: Icon(
+                  mutedAudio ? Icons.mic_off : Icons.mic,
+                  color: mutedAudio ? Colors.white : Colors.blueAccent,
+                  size: 20.0,
+                ),
+                shape: CircleBorder(),
+                elevation: 2.0,
+                fillColor: mutedAudio ? Colors.blueAccent : Colors.white,
+                padding: const EdgeInsets.all(12.0),
+              ),
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+
+  /// Info panel to show logs
+  /// Uncomment to test
+   Widget _panel() {
+     return Container();
+   }
+  //   return Container(
+  //     padding: const EdgeInsets.symmetric(vertical: 48),
+  //     alignment: Alignment.bottomCenter,
+  //     child: FractionallySizedBox(
+  //       heightFactor: 0.5,
+  //       child: Container(
+  //         padding: const EdgeInsets.symmetric(vertical: 48),
+  //         child: ListView.builder(
+  //           reverse: true,
+  //           itemCount: _infoStrings.length,
+  //           itemBuilder: (BuildContext context, int index) {
+  //             if (_infoStrings.isEmpty) {
+  //               return null;
+  //             }
+  //             return Padding(
+  //               padding: const EdgeInsets.symmetric(
+  //                 vertical: 3,
+  //                 horizontal: 10,
+  //               ),
+  //               child: Row(
+  //                 mainAxisSize: MainAxisSize.min,
+  //                 children: [
+  //                   Flexible(
+  //                     child: Container(
+  //                       padding: const EdgeInsets.symmetric(
+  //                         vertical: 2,
+  //                         horizontal: 5,
+  //                       ),
+  //                       decoration: BoxDecoration(
+  //                         color: Colors.yellowAccent,
+  //                         borderRadius: BorderRadius.circular(5),
+  //                       ),
+  //                       child: Text(
+  //                         _infoStrings[index],
+  //                         style: TextStyle(color: Colors.blueGrey),
+  //                       ),
+  //                     ),
+  //                   )
+  //                 ],
+  //               ),
+  //             );
+  //           },
+  //         ),
+  //       ),
+  //     ),
+  //   );
+  // }
```

Comment:
You should remove this before merging. It's fine for testing, but I don't think it should be in main.